### PR TITLE
Build /managed/cassandra

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new `/managed/cassandra` page
- Add to secondary nav 
- Add contact form at `shared/forms/interactive/managed-cassandra`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed/cassandra
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check page against [copy doc](https://docs.google.com/document/d/1iYMH1ISY4uDtr3ZzH0f_hnEzvqaeufO43z__cMxs0vQ/edit?ts=5f71f2cb#heading=h.xj070le8yswb)
- Check modal form against [form doc](https://docs.google.com/document/d/11QCRbr4aqRGEYXhU-9ob3Y4tYYLhrubJWimiaYFRVco/edit#)

## Issue / Card

Fixes [#3360](https://github.com/canonical-web-and-design/web-squad/issues/3360)

### CODE REVIEW

File: templates/managed/cassandra.html [hunk 1]
```diff
@@ -0,0 +1,659 @@
...
+<section class="p-strip--suru is-dark">
+  <div class="row u-vertically-center">
+    <div class="col-7">
+      <h1>
+        Managed Apache Cassandra
+      </h1>
+      <p><i>Now running Apache Cassandra 3.11</i></p>
+      <p class="p-heading--four">Fully managed Cassandra for your mission-critical data needs</p>
+      <p>
+        <a href="/managed/contact-us" class="p-button--positive">Get in touch</a>
```
Comment (anthonydillon) [inline templates/managed/cassandra.html:19]: Should we have the dynamic contact us modal class here?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Code LGTM with a me but there are some QA issues:
- On small screens the images seem huge:
image
- Can you update the link to Cassandra on /managed
- You can probably drop the icons

**Comment by SirSamTumless** [comment]
@anthonydillon the icons are the same as https://ubuntu.com/managed/kafka. Can we leave them for now as this needs to be pushed live ASAP and if we change them here, we'd need to change them on https://ubuntu.com/managed/kafka.

**Comment by bethcollins92** [comment]
@SirSamTumless @anthonydillon I'll just hide them on small screens?

File: templates/managed/cassandra.html (hunk 1 above)
Comment (SirSamTumless) [reply templates/managed/cassandra.html:19]: As before - this matches the Kafka page - so I don't really want to go chaning it.

Comment (bethcollins92) [reply templates/managed/cassandra.html:19]: Do you mean the same modal as the 'free deployment assessment' link next to it?

**Comment by SirSamTumless** [comment]
@bethcollins92 Sure, but can you make sure to do the same for https://ubuntu.com/managed/kafka

**Comment by bethcollins92** [comment]
Made changes and ready for re-review

**Comment by SirSamTumless** [comment]
@bethcollins92 LGTM 👍

**Review by SirSamTumless — APPROVED** [review]

**Pull request merged**